tf-modules/cumulus/variables.tf

```diff
@@ -626,3 +626,9 @@ variable "report_sns_topic_subscriber_arns" {
   default = null
   description = "Account ARNs to supply to report SNS topics policy with subscribe action"
 }
+
+variable "sf_event_sqs_lambda_timeout" {
```

Comment:
@Nnaga1  that's more or less what Jenny asked.     I have strong reservations about if we should really lump internal lambdas into a generic user configurable timeout given it's difficult to document how modifications to this could cause serious negative impacts and/or support obligations, but I'm willing to go with the team's opinion on it.

Reply:
Apologies for missing the back half of this conversation. I thought we'd agreed to NOT include this as part of the lambda_timeouts. The sfevent lambda is a central framework component that (ideally) we don't want users configuring in this way if they don't have to. At least that's my impression. I'm not sure what the consequences could be if this value changes.

Having this config in a separate, explicit, mildly buried variable for only users that really really know what they're doing was appealing. Giving the keys over in an object like lambda_timeouts makes it easier to see and adjust, which I'm not sure is what we want?

Reply:
tl;dr I kind of liked the initial approach and am a little concerned about this one but, like Jonathan, can be persuaded 😄

Reply:
For posterity: discussed this in a huddle on 4 October and agreed  with the current PR + scoped work to address and a arch meeting topic to discuss the approach with the team.